migrations/addProductSku_0001.sql

```diff
@@ -0,0 +1,2 @@
+IF NOT EXISTS( SELECT NULL FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'Products'AND table_schema = 'store' AND column_name = 'sku') THEN ALTER TABLE `Products` ADD `sku` VARCHAR(255);
```

Comment:
Since we now have this history aspect of migration runner, it's less important we keep these migrations idempotent. We can just make this:
```sql
ALTER TABLE `products` ADD `sku` VARCHAR(255);
```